docs/source/guides/localisation.rst

```diff
@@ -1,14 +1,101 @@
+            "MENU_TITLE_MODS" "Mods"
+            "RELOAD_MODS" "Reload Mods"
+            "WARNING" "Warning"
+            "CORE_MOD_DISABLE_WARNING" "Disabling core mods can break your client!"
+            "DISABLE" "Disable"
+        }
+    }
+
+It begins with the ``"lang"`` instruction, contains a ``"Language"`` key indicating language of current file's translations, and 
+a ``"Token"`` key indexing all translations.
+
+.. warning ::
+    Translations files must use ``"UTF-16 LE"`` encoding.
+
+You'll have to create one file per supported language, and all your files must be named in a similar fashion.
+
+For example, Northstar translation files are named ``"northstar_client_localisation_english.txt"``, ``"northstar_client_localisation_french.txt"``, 
+``"northstar_client_localisation_german.txt"`` etc.
+
+You can import them from your ``mod.json`` manifesto this way:
+
+.. code-block:: json
+
+    {
+        "Localisation": [
+            "resource/northstar_client_localisation_%language%.txt"
+        ]
+    }
 
-Files to translate
-------------------
+.. note::
+    The ``"%language%"`` syntax allows VM to load up translations matching game language (e.g. an English client will automatically use 
+    ``"northstar_client_localisation_english.txt"`` file)
+
+Use translations in your code
+-----------------------------
+
+To translate UI elements like menus, you have to insert strings containing your translation keys, preceded by a ``#``.
+
+For example, to translate the "Launch Northstar" button on main menu, instead of calling:
+
+.. code-block:: javascript
+
+    AddComboButton( comboStruct, headerIndex, buttonIndex++, "Launch Northstar" )
+
+We'll use:
+
+.. code-block:: javascript
+
+    AddComboButton( comboStruct, headerIndex, buttonIndex++, "#MENU_LAUNCH_NORTHSTAR" )
+
+You can also use the ``Localize`` method client-side:
+
+.. code-block:: javascript
+
+    Localize( "#MENU_LAUNCH_NORTHSTAR" )
+
+Northstar translations
+----------------------
+
+Northstar adds new strings to the game which can be localised to match the language you are using on your Titanfall 2 installation.
 
-There are the two files that need to be translated, use the English version as a base. After you finish make sure that the files are encoded in `UTF-16 LE` before opening a Pull Request.
+They're all located in ``"Northstar.Client"`` mod: `Northstar localisation files on GitHub <https://github.com/R2Northstar/NorthstarMods/blob/main/Northstar.Client/mod/resource>`_
 
-1. `Northstar.Client/mod/resource/northstar_client_localisation_english.txt <https://github.com/R2Northstar/NorthstarMods/blob/main/Northstar.Client/mod/resource/northstar_client_localisation_english.txt>`_
-2. `Northstar.Custom/mod/resource/northstar_custom_english.txt <https://github.com/R2Northstar/NorthstarMods/blob/main/Northstar.Custom/mod/resource/northstar_custom_english.txt>`_
+.. note ::
+    Don't hesitate to submit a PR if you notice a typo!
```

Comment:
I would leave this out

Reply:
Fixed with 3b1c30c2df79dea7cf4f7e08f94238bb01adc0d7.